command/alloc_logs.go

```diff
@@ -186,22 +192,25 @@ func (l *AllocLogsCommand) Run(args []string) int {
 		return 1
 	}
 
-	var task string
-	if len(args) >= 2 {
-		task = args[1]
-		if task == "" {
-			l.Ui.Error("Task name required")
-			return 1
-		}
-
+	// if -task isn't provided fallback to reading the task name
```

Comment:
Nit-pick, we try to capitalize Comments

```suggestion
	// If -task isn't provided fallback to reading the task name
```